### ISSUE TITLE
Missing info from VM IMAGES documentation

### ISSUE THREAD

**Issue by danboid** [ltsp/ltsp#618]
https://ltsp.org/man/ltsp-image/

Has a section called VM IMAGES, more specifically its this section that I think is lacking:

```
"At the disk creation dialog, select "VMDK" type and "Fixed size", not "Dynamically allocated". Proceed with installing Debian on it."
```

The easy correction is it should say a Debian or Ubuntu based distro, not just Debian.

Does it matter if the VM is a BIOS or UEFI machine? From your instructions I'm presuming LTSP expects a BIOS machine image but I presume a UEFI image would work just as well?

I'm presuming we can format the VM disk image with any filesystem except ZFS or would ZFS work too? How about btrfs? I have no reason to want to use either of those in this scenario but someone might try it. Are there any other partitioning requirements, limitations or expectations for the VM image like would having an image with a separate /boot or /home partition work? "Proceed with installing Debian" seemed a bit too vague to me.

**Comment by alkisg** [comment]
Hi danboid! The previous sentence before your quoted one, is:

> As an example, let's suppose you create a VM in VirtualBox and call it "debian".

Since the example VM is called debian, I don't think it makes sense to say "install Debian or Ubuntu on it"...
I needed to mention a specific example to be able to give example commands like the one that follows:

> ln -rs ~/VirtualBox\ VMs/debian/debian-flat.vmdk /srv/ltsp/debian.img

It wouldn't be pretty to have alternative commands for any possible distribution or VM name there. Maybe I could reword it to something like "my-example-vm" if it's confusing though.

For the second part of your question, please check the ltsp discussions, it's been answered many times already.

**Comment by danboid** [comment]
Hi @alkisg 

I'm not so much bothered about the Ubuntu/Debian thing, that doesn't really mattter but the partition spec of the image is a key bit of info that the user shouldn't have to search the discussions for.

I don't want to spend hours setting up an image for LTSP only to discover its partitions aren't configured correctly so please add this info or move these docs to the wiki so we can edit it for you.

**Comment by alkisg** [comment]
Thanks, fixed in # 91c04c1.

> move these docs to the wiki so we can edit it for you.

The ltsp code is [on github](https://github.com/ltsp/ltsp). You can fork it and send a PR for any part of the code or the man pages that you think needs some optimization. Putting the man pages in the wiki will probably make things worse, not better.
